README.md

```diff
@@ -10,30 +10,39 @@
 
 # Eightshift Frontend Libs
 
-This lib is intended to work in correlation with [Eightshift Libs](https://github.com/infinum/eightshift-libs/) and [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate) by hosting all frontend parts for your project.
- 
-## It contains:
-- All node modules required for the webpack build.
-- Eslinter ruleset.
-- Stylelinter ruleset.
-- Babel ruleset.
-- A collection of useful frontend utility modules.
-- Gutenberg Block example.
-- Gutenberg Blocks initial setup for [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate).
-- Some cool custom components for Gutenberg.
-- Complete webpack build for [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate).
-- Helpers for dynamically create Gutenberg blocks.
+This library is intended to work with [Eightshift Libs](https://github.com/infinum/eightshift-libs/) library and [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate) by containing all frontend parts for your project.
+
+## Documentaiton
+For the full documentation please check this [link](https://infinum.github.io/eightshift-frontend-libs).
+
+## Prerequisites
+
+It's recommended that you know how to work with core editor components and if you know some React.js, as this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
```

Comment:
```suggestion
The knowledge of the core editor components and React.js is recommended since this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
```

Reply:
👍